haas/cli.py

```diff
@@ -247,17 +247,24 @@ def serve_networks():
     """Start the HaaS networking server"""
     from haas import model, deferred
     from time import sleep
+    import ConfigParser
     server.init()
     server.register_drivers()
     server.validate_state()
     model.init_db()
     migrations.check_db_schema()
+    # Check if config contains usable sleep_time
+    try:
+        sleep_time = float(cfg.get('devel', 'sleep_time'))
+    except (ConfigParser.NoSectionError, ConfigParser.NoOptionError,
+            ValueError):
+        sleep_time = 2
```

Comment:
First, we should be using [getfloat()](https://docs.python.org/2/library/configparser.html#ConfigParser.RawConfigParser.getfloat) and not float().

Second, NoOptionError/NoSectionError are separate issues from ValueError. For the first two, the user didn't specify anything, so we should set a default for them.

For ValueError, two options would be to either throw an ERROR and exit in the exception block, or just remove the try/except and throw the exception directly back to the user. I like the first one better, since easy to understand error messages like `sleep_time set to non-integer value` is easier for a sysadmin to understand.

Finally, we should be doing a bound check (like 0 < sleep_time < 3600) on configured sleep values, printing an appropriate error message.

Thanks!